main.go

```diff
@@ -41,6 +43,12 @@ func main() {
 			},
 		},
 		Flags: []cli.Flag{
+			&cli.BoolFlag{
```

Comment:
Figured might as well. It's not set when Ansible invokes it, but when "shit goes wrong" we have a way to add `--debug` cause then it's interactive anyway. ;)